modules/widgets/wordpress-post-widget.php

```diff
@@ -238,14 +822,59 @@ public function form( $instance ) {
 		</p>
 
 		<?php
+
+		/**
+		 * Show error messages.
+		 */
+		if ( ! empty( $update_errors['message'] ) ) {
+
+			/**
+			 * Prepare the error messages.
+			 */
+
+			$where_message = __( 'An error occurred while downloading ', 'jetpack' );
```

Comment:
It's always best to put the full sentence/phrase/thought into a single translation call.  Can we rephrase this/these somehow?